src/loaders/OpenCellLoader.ts

```diff
@@ -0,0 +1,165 @@
+    const numChannels = 2;
+
+    // HQTILE or LQTILE
+    // make a json metadata dict for the two channels:
+    const urls = [
+      {
+        name: "https://opencell.czbiohub.org/data/opencell-microscopy/roi/czML0383-P0007/czML0383-P0007-A02-PML0308-S13_ROI-0424-0025-0600-0600-LQTILE-CH405.jpg",
+        channels: [0],
+      },
+      {
+        name: "https://opencell.czbiohub.org/data/opencell-microscopy/roi/czML0383-P0007/czML0383-P0007-A02-PML0308-S13_ROI-0424-0025-0600-0600-LQTILE-CH488.jpg",
+        channels: [1],
+      },
+    ];
+    // we know these are standardized to 600x600, two channels, one channel per jpg.
+    const chnames: string[] = ["DNA", "Structure"];
+
+    /* eslint-disable @typescript-eslint/naming-convention */
+    const imgdata: ImageInfo = {
+      width: 600,
+      height: 600,
+      channels: numChannels,
+      channel_names: chnames,
+      rows: 27,
+      cols: 1,
+      tiles: 27,
+      tile_width: 600,
+      tile_height: 600,
+      // for webgl reasons, it is best for atlas_width and atlas_height to be <= 2048
+      // and ideally a power of 2.  This generally implies downsampling the original volume data for display in this viewer.
+      atlas_width: 600,
+      atlas_height: 16200,
+      pixel_size_x: 1,
+      pixel_size_y: 1,
+      pixel_size_z: 2,
+      name: "TEST",
+      version: "1.0",
+      pixel_size_unit: "µm",
+      transform: {
+        translation: [0, 0, 0],
+        rotation: [0, 0, 0],
+      },
+      times: 1,
+    };
+    /* eslint-enable @typescript-eslint/naming-convention */
+
+    // got some data, now let's construct the volume.
+    const vol = new Volume(imgdata);
+    OpenCellLoader.loadVolumeAtlasData(vol, urls, onChannelLoaded);
+    return vol;
+  }
+
+    /**
+   * load per-channel volume data from a batch of image files containing the volume slices tiled across the images
+   * @param {Volume} volume
+   * @param {Array.<{name:string, channels:Array.<number>}>} imageArray
+   * @param {PerChannelCallback} onChannelLoaded Per-channel callback.  Called when each channel's atlased volume data is loaded
+   * @returns {Object.<string, Image>} a map(imageurl : Image object) that should be used to cancel the download requests,
+   * for example if you need to destroy the image before all data has arrived.
+   * as requests arrive, the callback will be called per image, not per channel
+   * @example loadVolumeAtlasData([{
+   *     "name": "AICS-10_5_5.ome.tif_atlas_0.png",
+   *     "channels": [0, 1, 2]
+   * }, {
+   *     "name": "AICS-10_5_5.ome.tif_atlas_1.png",
+   *     "channels": [3, 4, 5]
+   * }, {
+   *     "name": "AICS-10_5_5.ome.tif_atlas_2.png",
+   *     "channels": [6, 7, 8]
+   * }], mycallback);
+   */
+  static loadVolumeAtlasData(
+      volume: Volume,
+      imageArray: PackedChannelsImage[],
+      onChannelLoaded: PerChannelCallback
+    ): PackedChannelsImageRequests {
```

Comment:
So the OpenCell data just happens to be formatted as tiled rgb images, totally coincidental with our json format. Maybe better if OpenCellLoader calls into Json loader.  Otherwise I'm also kind of not worried if they stay separate.  OpenCellLoader is really not likely to be used by any production code -- in fact, maybe this loader could be moved into the public dir to live with the test app.
I'll have a look and refactor that.